samples/MauiEmbedding/.editorconfig

```diff
@@ -0,0 +1,167 @@
+; This file is for unifying the coding style for different editors and IDEs.
```

Comment:
I would probably avoid having more than one .editorconfig in the repo

Reply:
Done